t/022-redirect.t

```diff
@@ -318,5 +316,4 @@ Location: http://agentzh.org/foo?a=b&c=d
 GET /read
 --- response_headers
 Location: http://agentzh.org/foo?a=b&c=d
---- response_body_like: 308 Permanent Redirect
```

Comment:
Instead of removing these checks, I think it's better to adjust the test plan at the beginning of the test file by adding a 3 offset.

Reply:
@agentzh Now it should work just fine.